mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -105,6 +108,15 @@ class GaussianMixtureModel private[ml] (
   def gaussians: Array[MultivariateGaussian] = parentModel.gaussians
 
   @Since("2.0.0")
```

Comment:
@jkbradley @wangmiao1981 I vote ```GaussianMixtureModel.gaussians``` should return type of ```DataFrame``` for both Scala and Python, we need the same return type for different languages.
I refer that ```LDAModel.describeTopics``` and ```Word2VecModel.findSynonyms``` both return ```DataFrame``` in ML, but they use wrapper in MLlib such as [```describeTopics```](https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/api/python/LDAModelWrapper.scala#L34). I think return ```DataFrame``` for this kind of return value is good idea. I'm open to hear your thoughts. Thanks!